components/isceobj/Sensor/SAOCOM_SLC.py

```diff
@@ -0,0 +1,449 @@
+        instrument.setChirpSlope(float(self._xmlFileParser.pulseBandwidth)/float(self._xmlFileParser.pulseLength))
+
+        instrument.setRangeSamplingRate(self._xmlFileParser.frg)
+
+        incAngle = 0.5*(self.nearIncidenceAngle[self._xemtFileParser.beamID] + self.farIncidenceAngle[self._xemtFileParser.beamID])
+        instrument.setIncidenceAngle(incAngle)
+        
+    def _populateFrame(self):
+        """Populate the scene object with metadata"""
+        
+        rft = self._xmlFileParser.rangeStartTime
+        slantRange = float(rft)*Const.c/2.0
+        self.frame.setStartingRange(slantRange)
+
+        sensingStart = self._parseNanoSecondTimeStamp(self._xmlFileParser.azimuthStartTime)
+        sensingTime = self._xmlFileParser.lines/self._xmlFileParser.prf
+        sensingStop = sensingStart + datetime.timedelta(seconds=sensingTime)
+        sensingMid = sensingStart + datetime.timedelta(seconds=0.5*sensingTime)
+
+        self.frame.setPassDirection(self._xmlFileParser.orbitDirection)
+        self.frame.setProcessingFacility(self._xemtFileParser.facilityID)
+        self.frame.setProcessingSoftwareVersion(self._xemtFileParser.softVersion)
+        self.frame.setPolarization(self._xmlFileParser.polarization)
+        self.frame.setNumberOfLines(self._xmlFileParser.lines)
+        self.frame.setNumberOfSamples(self._xmlFileParser.samples)
+        self.frame.setSensingStart(sensingStart)
+        self.frame.setSensingMid(sensingMid)
+        self.frame.setSensingStop(sensingStop)
+
+        rangePixelSize = self.frame.getInstrument().getRangePixelSize()
+        farRange = slantRange +  (self.frame.getNumberOfSamples()-1)*rangePixelSize
+        self.frame.setFarRange(farRange)
+        
+    def _populateOrbit(self):
+        orbit = self.frame.getOrbit()
+        orbit.setReferenceFrame('ECR')
+        orbit.setOrbitSource('Header')
+        t0 = self._parseNanoSecondTimeStamp(self._xmlFileParser.orbitStartTime)
+        t = np.arange(self._xmlFileParser.numberSV)*self._xmlFileParser.deltaTimeSV
+        position = self._xmlFileParser.orbitPositionXYZ
+        velocity = self._xmlFileParser.orbitVelocityXYZ
+
+        for i in range(0,self._xmlFileParser.numberSV):
+            vec = StateVector()
+            dt = t0 + datetime.timedelta(seconds=t[i])
+            vec.setTime(dt)
+            vec.setPosition([position[i*3],position[i*3+1],position[i*3+2]])
+            vec.setVelocity([velocity[i*3],velocity[i*3+1],velocity[i*3+2]])
+            orbit.addStateVector(vec)
+            print("valor "+str(i)+": "+str(dt))
+        
+    def _populateExtras(self):
+        from isceobj.Doppler.Doppler import Doppler
+        
+        self.dopplerRangeTime = self._xmlFileParser.dopRngTime
+        self.rangeRefTime = self._xmlFileParser.trg
+        self.rangeFirstTime = self._xmlFileParser.rangeStartTime
+        
+    def extractImage(self):
+        """
+        Exports GeoTiff to ISCE format.
+        """
+        import gdal
```

Comment:
I would change this to 
```
from osgeo import gdal
```

Reply:
> @asolarte - thank you so much for this contribution. Just to understand - this is for the stripmap only correct?
> 
> Also, to let you know - if you add an unpackSAOCOM.py script to contrib/stack/stripmapStack folder - you will be able to generate stacks of coregistered SLCs as well with the stack processor.

@piyushrpt  - Yes, It's for stripmap only.
Thank you so much... I'll work on the unpackSAOCOM.py script